gsitemap.php

```diff
@@ -488,6 +494,131 @@ protected function getCategoryLink(&$link_sitemap, $lang, &$index, &$i, $id_cate
         return true;
     }
 
+    /**
+	 * return the link elements for the manufacturer object
+	 *
+	 * @param array  $link_sitemap    contain all the links for the Google Sitemap file to be generated
+	 * @param string $lang            language of link to add
+	 * @param int    $index           index of the current Google Sitemap file
+	 * @param int    $i               count of elements added to sitemap main array
+	 * @param int    $id_manufacturer manufacturer object identifier
+	 *
+	 * @return bool
+	 */
+	protected function getManufacturerLink(&$link_sitemap, $lang, &$index, &$i, $id_manufacturer = 0)
+	{
+		$link = new Link();
```

Comment:
You should be able to retrieve link from the context? :thinking:

Reply:
Done